src/hotspot/cpu/arm/macroAssembler_arm.cpp

```diff
@@ -673,15 +674,11 @@ void MacroAssembler::mov_metadata(Register rd, Metadata* o, int metadata_index)
 
 void MacroAssembler::mov_float(FloatRegister fd, jfloat c, AsmCondition cond) {
   Label skip_constant;
-  union {
-    jfloat f;
-    jint i;
-  } accessor;
-  accessor.f = c;
+  jint accessor_i = PrimitiveConversions::cast<jint>(c);
```

Comment:
I don't think the "accessor_" prefix has any meaning with this change, and just seems odd.
Just call the variable "bits" or "float_bits" or something like that?